pepys_admin/admin_cli.py

```diff
@@ -266,7 +269,13 @@ def do_export_reference_and_metadata_data(self):
                 self.data_store.db_classes.Privacy.name,
             ).all()
             privacy_dict = {name: privacy_id for privacy_id, name in privacies}
-        selected_privacies = iterfzf(privacy_dict.keys(), multi=True)
+        message = (
+            "Export all data with the selected classification(s). (Press TAB) for multi-select >"

```

Comment:
Can you just fix the brackets here, so it is `(Press TAB for multi-select)`?

Reply:
Fixed:
https://github.com/debrief/pepys-import/pull/417/commits/cfddc0bc3625262fe6df9a0e97949c3106eae214